assets/src/styles/campaigns/themes/_theme_antarctic.scss

```diff
@@ -18,6 +18,7 @@ body.theme-antarctic {
     line-height: 50px;
   }
 
+  font-family: var(--campaign_body_font, $lora);
```

Comment:
Can't we set `lora` through a css variable for the theme instead of in these files? Then the css declaration could be shared among all themes.

Reply:
We could do that here https://github.com/greenpeace/planet4-styleguide/blob/d41bfb0987ba4206129a0609a58deb5afef4d118/src/base/_body.scss#L8

Reply:
Hey @Inwerpsel  that was my original intention, but as some themes have a different *default* body font (e.g.: Climate has Jost, Forest has Roboto), I want to see the outcome of the P4CG body font ticket first (can't find it right now...).